testing/mocker/create_common.py

```diff
@@ -0,0 +1,150 @@
+
+        # Append all possible bids keywords
+        for keyword in bids_details.keys():
+            if "MODALITY" in keyword:
+                stem += f"_{bids_details.get('MODALITY', '')}"
+            elif "LABELS" in keyword:
+                stem += f"_{bids_details.get('LABELS', '')}"
+            else:
+                stem += f"_{keyword}-{bids_details.get(keyword,'')}"
+        return stem
+
+    def reject_missing_modalities(self, list_subject_specific_bids_dicts: List[dict]):
+        """
+        Validate the list of anatomy modalities.
+        :param list_subject_specific_bids_dicts:
+        :return:
+        """
+        for subject_specific_bids_dict in list_subject_specific_bids_dicts:
+            if "MODALITY" not in subject_specific_bids_dict:
+                logger.critical(f"MODALITY is not present in the subject dictionary provided: {pprint.pformat(subject_specific_bids_dict)}")
+                raise ValueError(
+                    f"Missing MODALITY field in the provided subject_specific_bids_dict."
+                )
+    def reject_missing_labels(self, list_subject_specific_bids_dicts: List[dict]):
+        """
+        Validate the list of anatomy modalities.
+        :param list_subject_specific_bids_dicts:
+        :return:
+        """
+        for subject_specific_bids_dict in list_subject_specific_bids_dicts:
+            if "LABELS" not in subject_specific_bids_dict:
+                logger.critical(f"LABELS is not present in the subject dictionary provided: {pprint.pformat(subject_specific_bids_dict)}")
+                raise ValueError(
+                    f"Missing LABELS field in the provided subject_specific_bids_dict."
+                )
+    def generate_file_description_dict(self, bids_detail: dict) -> dict:
+        """
+        Fill a json dictionary with a description of the nifti image.
+        :param bids_detail:
+        :return:
+        """
+        mock_meta_data = {
+            "Modality": "MR",
+            "MagneticFieldStrength": 3,
+            "Manufacturer": "MOCK",
+            "InstitutionName": "MOCK Research Center",
+            "MRAcquisitionType": "3D",
+            "SeriesDescription": bids_detail.get("MODALITY", ""),
+            "ProtocolName": bids_detail.get("MODALITY", ""),
+            "EchoTime": 1,
+            "RepetitionTime": 1,
+            "InversionTime": 1,
+            "FlipAngle": 1,
+            "SliceThickness": 1,
+            "ConversionSoftware": "MOCK",
+        }
+        return mock_meta_data
+
+    def generate_dataset_description_dict(self) -> dict:
+        """
+        Fill a json dictionary with a description of the entire dataset of the collections of the nifti images.
+        :return:
+        """
+        mock_meta_dataset = {
+            "Name": "MOCK multi-permuted_bids_detail multi-session dataset",
+            "BIDSVersion": "1.0.2",
+            "Researcher": "MOCK_RESEARCHER",
+            "Study": "MOCK_STUDY",
```

Comment:
```suggestion
            "Authors": "MOCK_AUTHOR",
```
Similar to my previous comment about JSON fields.
"Researcher" is not a supported field and can be replaces by "Authors".
"Study" doesn't exist and doesn't have a direct correspondence.
Ref [here](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#dataset_descriptionjson).